server-base/src/main/java/org/apache/kylin/rest/response/SQLResponse.java

```diff
@@ -205,20 +209,42 @@ public String getTraceUrl() {
     public void setTraceUrl(String traceUrl) {
         this.traceUrl = traceUrl;
     }
-    
+
+    @JsonIgnore
+    public String getSignature() {
+        return signature;
+    }
+
+    public void setSignature(String signature) {
+        this.signature = signature;
+    }
+
+    @JsonIgnore
+    public long getDummyTime() {
```

Comment:
can we have a meaningful name rather than dummy? this is hard to understand what does it do.